.github/workflows/markdown-lint.yaml

```diff
@@ -0,0 +1,18 @@
+name: Markdown-lint
+on:
+  pull_request:
+
+jobs:
+  lint-changelog:
+    name: Lint changelog file
+    runs-on: ubuntu-latest
+    steps:
+    - name: Check out code
+      uses: actions/checkout@v2
+
+    - name: Lint changelog file
+      uses: docker://avtodev/markdown-lint:v1
+      with:
+        rules: '/lint/rules/changelog.js'
+        config: '/lint/config/changelog.yml'
```

Comment:
where are these files created?

Reply:
these files are not required in our case. These files are config files for the linter.  Initially, I thought these are required fields. Thanks